test/jest/resources/versions.ts

```diff
@@ -1,4 +1,6 @@
-export default {
+import type { VersionResponse } from '../../../backend/handler/AuthHandler.types';
+
+const versinos: Record<string, VersionResponse> = {
```

Comment:
```suggestion
const versions: Record<string, VersionResponse> = {
```

and also L24 needs to be corrected.

Reply:
I fixed the typo: [40a7188](https://github.com/cloudify-cosmo/cloudify-stage/pull/2247/commits/40a7188af180dd380ad0d777b6d5d1beb17402c7)